cmake/leatherman.cmake

```diff
@@ -59,6 +59,26 @@ macro(defoption name doc default)
     option(${name} ${doc} ${enabled})
 endmacro()
 
+# Usage: leatherman_install(TARGETS)
+#
+# Installs targets using common cross-platform configuration.
+# On Windows shared libraries go in bin, import and archive libraries
+# go in lib. On Linux shared libraries go in lib. Binaries go in bin.
+#
+# Also always drop the prefix; give the target its expected name.
+# We often have binaries and related dynamic libraries, and this
+# simplifies giving them different but related names, such as
+# `facter` and `libfacter`.
+macro(leatherman_install)
+    install(TARGETS ${ARGV}
+        RUNTIME DESTINATION bin
```

Comment:
I think these RUNTIME, LIBRARY, and ARCHIVE destinations are all the CMake defaults?